backend/apid/graphql/mutations.go

```diff
@@ -86,12 +86,12 @@ func (r *mutationsImpl) DeleteCheck(p schema.MutationDeleteCheckFieldResolverPar
 	ctx := setContextFromComponents(p.Context, components)
 
 	client := r.factory.NewWithContext(ctx)
-	check, err := client.FetchCheck(components.UniqueComponent())
+	_, err := client.FetchCheck(components.UniqueComponent())
```

Comment:
So, since we don't need the check returned by `FetchCheck()` anymore, we are purely making a call to that function for the side effect it has of checking if a check actually exists?